packages/core-components/src/components/button/button.scss

```diff
@@ -0,0 +1,9 @@
+@import '../../../../odyssey/src/scss/base/reset';
+@import '../../../../odyssey/src/scss/base/typography-link';
+@import '../../../../odyssey/src/scss/components/button';
+
+// Adding width to inherit from parent to support @okta/admin-app
+// search bar which is created using o-button to span whole width
+.ods-button {
+  width: inherit;
+}
```

Comment:
This file shouldnt exist, and instead should come straight from Odyssey. If there is a reason for having this CSS _which all consumers care about_ we should integrate it directly into `@okta/odyssey` instead of doing it this way.

Otherwise, this code should live within `@okta/admin-app`